### ISSUE TITLE
Bug in heading anchors

### ISSUE THREAD

**Issue by jennet** [digital-land/specification#32]
Sorry if this is the wrong repo to add this (very minor!) issue, but I was just trying to grab a deep link to a particular heading on https://digital-land.github.io/specification/specification/development-plan/ and the code is showing 
```
<h3 id="{d[" dataset"]{}}-dataset"="" class="govuk-heading-m">
    Development plan
</h3>
```

 so I can't grab the ID in order to link to it.

**Comment by colmjude** [comment]
Hi @jennet - thanks for the heads up! You are definitely in the right place!

We've fixed the issue with commit 62c0397

**Comment by jennet** [comment]
perfect yep that's sorted it!
